src/pseudopeople/configuration/validator.py

```diff
@@ -54,12 +54,20 @@ def validate_overrides(overrides: Dict, default_config: ConfigTree) -> None:
             default_noise_type_config = _get_default_config_node(
                 default_row_noise_config, noise_type, "noise type", dataset
             )
+            parameter_config_validator_map = {
```

Comment:
Oof, this is getting contrived. This makes me feel even stronger like this function is kind of like a third noise type with a different set of rules.

Reply:
Wait...why can't you just add this directly to the DEFAULT_PARAMETER_CONFIG_VALIDATOR_MAP?